android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginScreen.kt

```diff
@@ -404,7 +406,10 @@ fun ForgotPasswordDialog(
       )
     },
     text = {
-      Text(text = stringResource(R.string.call_supervisor, "012-3456-789"), fontSize = 16.sp)
+      Column(modifier = modifier.fillMaxWidth().padding(horizontal = 16.dp, vertical = 12.dp)) {
+        Text(text = stringResource(R.string.please_contact_supervisor), fontSize = 16.sp)
```

Comment:
We can default to the previous message if the supervisor contact is null, empty, or black otherwise display their configured contact.